x-pack/plugins/security/public/management/roles/edit_role/privileges/kibana/feature_table/sub_feature_form.tsx

```diff
@@ -130,6 +131,15 @@ export const SubFeatureForm = (props: Props) => {
             props.onChange([...privilegesWithoutGroupEntries, selectedPrivilegeId]);
           }
         }}
+        legend={i18n.translate(
+          'xpack.security.management.editRole.subFeatureForm.controlLegendText',
+          {
+            defaultMessage: '{subFeatureName} feature privilege',
```

Comment:
Maybe this is a bit pedantic, but a Feature Privilege is distinct from a Sub-feature Privilege. I worry that using the same word in the legend to describe both will be confusing for screen reader users. I think the safest thing to do is to change this:
```suggestion
            defaultMessage: '{subFeatureName} sub-feature privilege',
```

Reply:
Agh, I see that you suggested this originally. Will update